src/main/steps/c100-rebuild/c100sequence.ts

```diff
@@ -444,4 +445,12 @@ export const C100Sequence: Step[] = [
     showInSection: Sections.C100,
     getNextStep: () => C100_CONFIDENTIALITY_DETAILS_KNOW,
   },
+  {
+    url: C100_C1A_SAFETY_CONCERNS_OTHER_CONCERNS_DRUGS,
+    showInSection: Sections.C100,
+    getNextStep: (data: Partial<Case>) =>
+      data.otherConcernsDrugs === YesOrNo.YES
```

Comment:
@niranjan-301 Looking at the Miro & story details, there is no conditional logic for navigation for this screen. Not sure why you added conditions for this.